packit_service/worker/helpers/build/copr_build.py

```diff
@@ -537,10 +596,62 @@ def submit_copr_build(self, script: Optional[str] = None) -> Tuple[int, str]:
 
         return build.id, self.api.copr_helper.copr_web_build_url(build)
 
-    def handle_build_submit_error(self, ex) -> TaskResults:
+    def run_copr_build(self) -> TaskResults:
```

Comment:
I'm a little confused here; if I am checking correctly, this method was removed already, maybe a mistake when rebasing?